lib/sidekiq/statistic/base.rb

```diff
@@ -31,7 +31,7 @@ def statistic_hash
 
       def get_statistic_hash(conn, redis_hash)
         conn
-          .hgetall(REDIS_HASH)
+          .hgetall(Metrics::Store::REDIS_HASH)
```

Comment:
This module as well as the `realtime.rb` one uses the `REDIS_HASH` in order to read Redis data, in future refactoring I believe it worths to isolate those `read` operations in a single/specialized module.